src/components/Button/Button.jsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
-
+import PropTypes from 'prop-types';
 import styles from './Button.scss';
 
-function Button({ clickHandler }) {
+function Button({ onClick }) {
   return (
-    <button type="button" className={styles['menu-btn']} onClick={clickHandler}>
+    <button type="button" className={styles.menuBtn} onClick={onClick}>
       Menu
     </button>
   );
 }
 
+Button.propTypes = {
+  onClick: PropTypes.func
```

Comment:
what will happen if the consumer of your Button doesn't pass onClick handler ?

hint: the code will fail. You gotta define .Required prop or a default prop for onClick